src/database.rs

```diff
@@ -262,7 +262,7 @@ impl Database {
                 if index_path.exists() {
                     trace!("Database.build_keydir: index found 'index.{}'. Importing data file No={} Path={} ...", file_id, file_id, &entry.display());
 
-                    let mut index = IndexFile::create(&index_path.to_path_buf(), true).unwrap();
```

Comment:
this is a redundant clone. `IndexFile::create` takes in a path slice (`&Path`) which borrowing a `PathBuf` produces.

this was cloning the `PathBuf` just to borrow it as a slice.